### PR TITLE
WebUI: Refine use of Datepicker widgets #4184

### LINKED ISSUE #4184: Suspicious replicas page is misinterpreting the date picker
Motivation
----------

The code expects the return string from the date picker to be in `DD/MM/YYYY` format but it’s actually in `MM/DD/YYYY`.

Modification
------------

Swap the array indices.

### PR DESCRIPTION
**Pull request by dchristidis**
Continuation from #4186.

There are multiple pages with Datepicker widgets in the Web UI. I’ve had this branch sitting around for quite a while where one of them is fixed. Before I move on to the others, could I ask for a quick look for obvious flaws?

### CODE REVIEW

File: lib/rucio/web/ui/static/bad_replicas_summary.js [hunk 1]
```diff
@@ -51,57 +51,36 @@ function draw_pie(data) {
...
-                                      defaultDate: from_date,
+                                      defaultDate: default_from_date,
                                       minDate: new Date(2015, 0, 1),
-                                      onSelect: function(){
-                                          from_date = $("#datepicker1").val();
-                                          $("#datepicker2").datepicker('setDate', from_date).datepicker('option', 'minDate', from_date);
+                                      dateFormat: $.datepicker.ISO_8601,
+                                      onSelect: function(new_from_date) {
+                                          const new_min_to_date = new Date(new Date(new_from_date).getTime() + ms_per_day);
+                                          $('#datepicker2').datepicker('option', 'minDate', new_from_date);
```
Comment (dchristidis) [inline lib/rucio/web/ui/static/bad_replicas_summary.js]: `s/new_from_date/new_min_to_date/`

**Comment by mlassnig** [comment]
@dchristidis status?

**Comment by maany** [comment]
@dchristidis does this still require further work?

**Review by maany — APPROVED** [review]
btw, the code changes work well ;) Thanks @dchristidis

**Pull request merged**